dummy_lib/core.py

```diff
@@ -11,3 +13,21 @@ def greet_contributor(name: str) -> str:
 		the greeting message
 	"""
 	return f"Hello {name}! Nice to meet you."
+
+
+def feature_a(input_temperatures: List[float]) -> List[float]:
+	"""Converts temperatures from Fahrenheit to Celsius.
+
+	Args:
+		input_temperatures: the list of temperatures in Fahrenheit
+
+	Returns:
+		the temperatures converted to Celsius
+	"""
+
+	tmp = []
+	for fahrenheit_temp in input_temperatures:
+		celsius_temp = (fahrenheit_temp - 32) * 5 / 9
+		tmp.append(celsius_temp)
+
+	return tmp
```

Comment:
This section introduces: an unnecessary "for" loop, an unnecessary intermediate variable :confused: 

What about:

```suggestion
	return [(fahrenheit_temp - 32) * 5 / 9 for fahrenheit_temp in input_temperatures]
```

This feedback is about: **[performance] [compilation bottleneck]**. If this is wrong, click [here](url).